main.rb

```diff
@@ -33,13 +35,14 @@ def run
     when '3'
       @peoples.push(create_person)
     when '4'
-      @books.push(create_book)
+      create_book
     when '5'
       @rentals.push(create_rental(@books, @peoples.last))
       puts "\n"
```

Comment:
[OPTIONAL] Because this is not a requirement in this milestone but in a previous one the following is optional however I strongly suggest you change it as the application is not working as intended.
- [ ] when creating a rental the application asks for a book however it takes the last person which is created, however, according to the requirements in a previous milestone the user should be able to pick the person to which the book is rented by ID. please try implementing this.

Reply:
Thank you for the optional suggestion, It will be implemented by the next milestone.